src/content/basics/api/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: API Access to Giant Swarm Resources
+description: An overview of the APIs that provide you with programmatic access to
+  resources like your tenant cluster in a Giant Swarm installation. Namely the Control
+  Plane Kubernetes API and the legacy Giant Swarm Rest API.
+date: 2020-05-04
+weight: 75
+type: page
+categories: ["basics"]
+last-review-date:
+---
+
+# API Access to Giant Swarm Resources
+
+For integrating Giant Swarm with your automation, we provide programmatic access via two different APIs:
+
+- [The Control Plane Kubernetes API](#cp-k8s-api)
+
+- [The Giant Swarm Rest API](#rest-api) (deprecated)
+
+In the following sections we explain the differences and specific benefits and provide some guidance on how to get started.
+
+## The Control Plane Kubernetes API (CP-K8s-API) {#cp-k8s-api}
+
+## What it is
+
+At Giant Swarm, when we say "control plane", we talk about the Kubernetes Cluster that runs all the operational and monitoring workloads which are needed to create and manage the "tenant clusters", which are the clusters you create to run your actual workloads. This is not to be confused with how the Kubernetes project uses the same term, simply meaning the master nodes of a Kubernetes cluster.
+
+The control plane is a Kubernetes cluster. Your tenant clusters and other associated resources are represented in that cluster as custom resources. To access these, you can use the Kubernetes API of the cluster that forms the control plane, or in short, the Control Plane Kubernetes API. For the sake of brevity, we'll abbreviate this as _CP-K8s-API_.
+
+### How to gain access
+
+To access the CP-K8s-API, Giant Swarm configures access to the API via OIDC. Please contact Solution Engineer (SE) to sort out the details.
+
+Currently we normally provide read-only access by default. As we are currently working on a fine-grained way to control permissions to resources, having write access right now means fully unrestricted access and should only be granted to select individuals.
+
+TODO: Explain how setting up write is different from read and what else it requires.
+
+### How to use
+
+We recommend that you become familiar with the API via `kubectl`, navigating your resources via the CP-K8s-API. Besides general Kubernetes know-how this will require only a bit of structural knowledge:
```

Comment:
Can we condense this sentence a bit? Also could we recommend something lik eoctant here as a way to visually navigate and explore through the resources?

```suggestion
We recommend using `kubectl` to navigate the resources present on the CP-K8s-API. Besides general Kubernetes know-how this will require only a bit of structural knowledge:
```